sdks/how-to/how-to-manage-healthcare-elements.md

```diff
@@ -4,3 +4,283 @@ description: Learn how to manage healthcare elements
+const healthcareElement1 = new HealthcareElement({
+  description: 'The patient has been diagnosed Pararibulitis',
+  codes: new Set([
+    new CodingReference({
+      id: 'SNOMEDCT|617|20020131',
+      type: 'SNOMEDCT',
+      code: '617',
+      version: '20020131',
+    }),
+  ]),
+  openingDate: new Date('2019-10-12').getTime(),
+})
+
+const healthcareElement2 = new HealthcareElement({
+  description: 'The patient has also the flu',
+  openingDate: new Date('2020-11-08').getTime(),
+})
+
+const newElements = await api.healthcareElementApi.createOrModifyHealthcareElements(
+  [healthcareElement1, healthcareElement2],
+  patient.id,
+)
+```
+
+:::caution
+
+Even if you associate a Healthcare Element to a Patient, the Patient does not have access to it unless you give access 
+to them.
+
+:::
+
+
+
+## Sharing a Healthcare Element with a Patient
+
+After creating the Healthcare Element, the Healthcare Professional shares it with the Patient.
+
+<!-- file://code-samples/how-to/manage-healthcare-elements/index.mts snippet:HE sharing with data owner-->
+```typescript
+const sharedHealthcareElement = await api.healthcareElementApi.giveAccessTo(
+  healthcareElement,
+  patient.id,
+)
+```
+
+If the operation is successful, the method returns a Promise with the updated Healthcare Element.
+
+## Retrieving a Healthcare Element Using its ID
+
+A single Healthcare Element can be retrieved from the Backend using its id.
+
+<!-- file://code-samples/how-to/manage-healthcare-elements/index.mts snippet:retrieve a HE as data owner-->
+```typescript
+const healthcareElementToRetrieve = new HealthcareElement({
+  description: 'To retrieve, I must create',
+})
+
+const createdHealthcareElement = await api.healthcareElementApi.createOrModifyHealthcareElement(
+  healthcareElementToRetrieve,
+  patient.id,
+)
+
+const retrievedHealthcareElement = await api.healthcareElementApi.getHealthcareElement(
+  createdHealthcareElement.id,
+)
+```
+
+:::caution
+
+Trying to retrieve a Healthcare Element you do not have access to will produce an error.
+
+:::
+
+## Modifying a Healthcare Element
+
+Given an existing Healthcare Element, it is possible to modify it.  
+Some fields, like id and rev, cannot be modified.
+
+<!-- file://code-samples/how-to/manage-healthcare-elements/index.mts snippet:modify a HE as data owner-->
+```typescript
+const yetAnotherHealthcareElement = await api.healthcareElementApi.createOrModifyHealthcareElement(
+  new HealthcareElement({
+    description: 'To modify, I must create',
+  }),
+  patient.id,
+)
+
+const modifiedHealthcareElement = {
+  ...yetAnotherHealthcareElement,
+  description: 'I can change and I can add',
+  openingDate: new Date('2019-10-12').getTime(),
+}
+
+const modificationResult = await api.healthcareElementApi.createOrModifyHealthcareElement(
+  modifiedHealthcareElement,
+  patient.id,
+)
+```
+
+If the operation is successful, the method returns the updated Healthcare Element as it is stored in the Backend.
```

Comment:
If the operation is successful, the method returns the updated Healthcare Element.